test/unit/global-tagging.v1.test.js

```diff
@@ -17,7 +17,7 @@
 // need to import the whole package to mock getAuthenticatorFromEnvironment
 const sdkCorePackage = require('ibm-cloud-sdk-core');
 
-const { NoAuthAuthenticator } = sdkCorePackage;
+const { NoAuthAuthenticator, unitTestUtils } = sdkCorePackage;
```

Comment:
Need to re-gen with the most recent SDK generator version and these import-related changes should go away.

Reply:
Resolved by @padamstx